wdio.conf.js

```diff
@@ -0,0 +1,329 @@
+    //
+    // If you have trouble getting all important capabilities together, check out the
+    // Sauce Labs platform configurator - a great tool to configure your capabilities:
+    // https://saucelabs.com/platform/platform-configurator
+    //
+
+    capabilities: [
+        {
+            maxInstances: 1,
+            port: 4444,
+            browserName: 'chrome',
+            acceptInsecureCerts: true,
+            "goog:chromeOptions": {
+                args: ['use-fake-device-for-media-stream',
+                    'use-fake-ui-for-media-stream',
+                    'disable-plugins',
+                    'mute-audio',
+                    'disable-infobars',
+                    'autoplay-policy=no-user-gesture-required',
+                    'auto-select-desktop-capture-source=Your Entire screen'],
+            }
+        },
+        {
+            maxInstances: 1,
+            browserName: 'firefox',
+            acceptInsecureCerts: true,
+            "moz:firefoxOptions": {
+                "prefs": {
+                    "media.navigator.streams.fake": true,
+                    "media.navigator.permission.disabled": true,
+                    "media.peerconnection.ice.tcp": true,
+                    "intl.accept_languages": "en",
+                    "media.autoplay.default": 0
+                }
+            }
+        },
+        {
+            maxInstances: 1,
+            port: 4445,
+            browserName: 'safari technology preview',
+            acceptInsecureCerts: true,
+        }
+    ],
+    //
+    // ===================
+    // Test Configurations
+    // ===================
+    // Define all options that are relevant for the WebdriverIO instance here
+    //
+    // Level of logging verbosity: trace | debug | info | warn | error | silent
+    logLevel: 'info',
+    //
+    // Set specific log levels per logger
+    // loggers:
+    // - webdriver, webdriverio
+    // - @wdio/browserstack-service, @wdio/devtools-service, @wdio/sauce-service
+    // - @wdio/mocha-framework, @wdio/jasmine-framework
+    // - @wdio/local-runner
+    // - @wdio/sumologic-reporter
+    // - @wdio/cli, @wdio/config, @wdio/utils
+    // Level of logging verbosity: trace | debug | info | warn | error | silent
+    // logLevels: {
+    //     webdriver: 'info',
+    //     '@wdio/appium-service': 'info'
+    // },
+    //
+    // If you only want to run your tests until a specific amount of tests have failed use
+    // bail (default is 0 - don't bail, run all tests).
+    bail: 0,
+    //
+    // Set a base URL in order to shorten url command calls. If your `url` parameter starts
+    // with `/`, the base url gets prepended, not including the path portion of your baseUrl.
+    // If your `url` parameter starts without a scheme or `/` (like `some/path`), the base url
+    // gets prepended directly.
+    baseUrl: 'http://localhost',
+    //
+    // Default timeout for all waitFor* commands.
+    waitforTimeout: 10000,
```

Comment:
IMHO every waitFor should be explicit, waiting for a long time everywhere sounds not so great.